gnowledge/gstudio
# Phase - I Implemented Memcached

**PHASE-I Memcached implementated**
- Some of the time-costly views cached according to different parameters.
- Most of the redundant templates are cached.
- So mostly memcached implemented at template and view level.
- There is much more to do in the memcached implementation. It's trade-off between **Time**, **Frequency**, Memory-space**. The decision needs to be taken from/after phase-I implementation. So I think it could be parallel process to development. It will get mature with time.

--

**INSTALL TO-DO**
- `sudo apt-get install libevent-dev`
- `sudo apt-get install memcached`
- `pip install python-memcached`
- `pip install django-memcache-admin`

--

**TO TEST**
- Use more than one browser session/different-browsers. In one try with one user login, in another with other user login and in other/third try to use as anonymous user. And check the headers and other parts of same page opened in different browser/sessions.
- Use firebug/Browser-Inspector to lookout for performance.
